### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by Dayveed12345**
Made The version for the composer.json require key to "^7.3" instead of 8.1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^8.1",
```
Comment (milwad-dev) [inline composer.json:19]: https://github.com/illuminate/contracts/blob/master/composer.json#L17

**Comment by Dayveed12345** [comment]
It was 8.2 initially and you changed it to 8.1 hope it won't have any
effect on the source code

File: composer.json (hunk 1 above)
Comment (Dayveed12345) [reply composer.json:19]: Okay thanks

**Comment by milwad-dev** [comment]
If you look at @driesvints PR, you can see Laravel 11.x support PHP 8.2, now Laravel 10.x work with PHP 8.1

https://github.com/illuminate/contracts/commit/e81b81639b2f757c46eca1d0e252963ac8a0fe78

**Comment by Dayveed12345** [comment]
Okay so we are working with Laravel 10

**Comment by Dayveed12345** [comment]
I was a actually going through c# windows form Nd was thinking of
implementing some properties like that will that be okay?

**Comment by simonhamp** [comment]
There are code features here which require PHP 8.1. Also PHP 7.3 is well past its end of life, so we won't be supporting that

**Comment by Dayveed12345** [comment]
Okay interesting!!
 Concerning the c# windows form features

**Comment by Dayveed12345** [comment]
No response

**Comment by Dayveed12345** [comment]
I am getting this error when am trying to use the composer require
nativephp/electron

**Pull request closed without merging**